### ISSUE TITLE
Missing app.py?

### ISSUE THREAD

**Issue by FrancisCorona** [samm-o/Sports-Betting-Predictor#1]
I couldn’t locate the app.py file to run the backend. Is the file called something else or am I missing something? Thank you!

**Comment by samm-o** [comment]
Hey! Thanks for bringing this up! So it's actually ConfidenceScores.py that runs the flask backend server. Then you can cd into nfl-predictions and run npm start.

**Comment by FrancisCorona** [comment]
Gotcha! Thank you for getting back to me! I can't seem to find the ConfidenceScores.py file. I see ConfidenceScores in the Parsing directory though.

**Comment by samm-o** [comment]
Yeah exactly, within the Parsing directory, install the dependencies (flask, flask_cors) then run it. Then you'll see a flask app deploy in which the node server will run on top of.

**Comment by FrancisCorona** [comment]
Awesome, Thank you! I was able to get it running! 

What do the confidence numbers mean? I see the explanation of where they come from, but am unsure of how to apply them. Can you please advise?

**Comment by samm-o** [comment]
Awesome! Yeah, it is a little unclear I will update that. But the confidence scores mean the confidence of the model of the Home team winning (left side team)

So for example, if you look at the 1st game, KC vs BAL, They all have a confidence score of over 50% which means they all favour KC to win rather than BAL. The confidence is how sure they are, so 100% means the model is very sure KC (home) will win and 0% means the mode is very sure that BAL (Away) will win.

**Comment by FrancisCorona** [comment]
Awesome! Thank you for all your help!!

**Comment by kodekid32** [comment]
Can you provide update on how to re-scrape data for update predictions? Can't seem to figure out how to get the excel sheets to update.
